readme.md

```diff
@@ -36,7 +37,20 @@ $ ncc run input.js
 
 Build to a temporary folder and run the built JS file through Node.js, with source maps support for debugging.
 
-### Node.js
+### With TypeScript
+
+The only requirement is to point `ncc` to `.ts` or `.tsx` files. A `tsconfig.json`
+file is necessary. Most likely you want to indicate `es2015` support:
+
+```json
+{
+  "compilerOptions": {
+    "target": "es2015"
+  }
+}
+```
```

Comment:
I wonder if we could make the `tsconfig.json` optional with the above as the default?

Reply:
Good observation. It's not recommended due to editor integration. In other words, e.g.: VSCode would need a mechanism to understand what rules `ncc` would be using.

It's a small extra step, but keeps all tooling in sync. Plus, depending on what Node.js version the developer is targeting, the `target` can be adjusted, which is quite nice.

Reply:
Makes sense 👍